src/command_modules/azure-cli-container/azure/cli/command_modules/container/custom.py

```diff
@@ -230,7 +230,7 @@ def _get_vnet_network_profile(cmd, location, resource_group_name, vnet_name, vne
     containerInstanceDelegationName = "Microsoft.ContainerInstance.containerGroups"
     Delegation = cmd.get_models('Delegation', resource_type=ResourceType.MGMT_NETWORK)
     aci_delegation = Delegation(
-        name="Microsoft.ContainerInstance.containerGroups",
+        name="Microsoft.ContainerInstance/containerGroups",
```

Comment:
Pretty sure the `name` here was correct to begin with.  You've changed it to the `service_name`.